ui/src/app/utils/index.spec.tsx

```diff
@@ -4,18 +4,6 @@ import * as Utils from 'app/utils';
 import { usernameWithoutDomain } from 'app/utils';
 
 describe('Helper functions', () => {
-  it('(cond) Should handle conditionals and defaults correctly', async () => {
-    const test = (x) =>
-      Utils.cond<number | string>(
-        [x === 5, () => 1],
-        [x === 10, () => 2],
-        () => 'default'
-      );
-    expect(test(5)).toBe(1);
-    expect(test(10)).toBe(2);
-    expect(test(9)).toBe('default');
-  });
-
```

Comment:
This function is [well-tested within the Terra UI codebase](https://github.com/DataBiosphere/terra-ui/blob/c79428c00ec8c32994294257ffb628e3d1d9b10f/packages/core-utils/src/logic-utils.test.ts), so removing this now-redundant test.